routes.js

```diff
@@ -24,7 +25,7 @@ router.get('/', auth, (req, res) => {
 router.get('/user/', auth, (req, res) => userController.retrieveAll(req, res));
 router.post('/user/', auth, (req, res) => userController.create(req, res));
 router.get('/user/:id', auth, (req, res) => userController.retrieveById(req, res));
-router.post('/user/update/:id', auth, (req, res) => userController.update(req, res));
+router.patch('/user/update/:id', auth, (req, res) => userController.update(req, res));
```

Comment:
Why are you sending the `userId` from parameters and not the body like the rest of the parameters?